static/app/components/events/featureFlags/featureFlagInlineCTA.spec.tsx

```diff
@@ -0,0 +1,93 @@
+import {render, screen, userEvent} from 'sentry-test/reactTestingLibrary';
+
+import FeatureFlagInlineCTA from 'sentry/components/events/featureFlags/featureFlagInlineCTA';
+
+describe('featureFlagInlineCTA', () => {
+  beforeEach(() => {
+    MockApiClient.clearMockResponses();
+    MockApiClient.addMockResponse({
+      url: '/organizations/org-slug/prompts-activity/',
+      body: {data: {dismissed_ts: null}},
+    });
+  });
+
+  it('shows an onboarding banner that may be dismissed', async () => {
```

Comment:
we're missing some tests that make sure it doesn't render if 
1. platform not javascript or python (do we have a list of platforms anywhere we can test `platforms.filter(p => !platforms.includes(p))`
2. feature flag is on
3. the other condition there (the context.flags is undefined)?

`group.project.platform` maps to `event.platform`? seems to be project platform, which would map to more detailed stuff like `javascript-react` or something?

Reply:
1. will add a test for this

3. which feature flag are you referring to here? `feature-flag-ui` or the launchdarkly flag? if it's `feature-flag-ui`, the entire feature flag section doesn't render if the flag is false. see code snippet: 
https://github.com/getsentry/sentry/blob/40f788c6a2efff765a2d05a6c4f472b3d81b7d0a/static/app/views/issueDetails/groupEventDetails/groupEventDetailsContent.tsx#L423-L425

if the LD flag, we haven't defined it yet, but i can add it into the tests when it's defined. right now in the test i'll just test if the platform is `javascript` or `python`.

4. already implemented in a different test file: https://github.com/getsentry/sentry/blob/40f788c6a2efff765a2d05a6c4f472b3d81b7d0a/static/app/components/events/featureFlags/eventFeatureFlagList.spec.tsx#L216

`group.project.platform` is the same as `event.platform` and `project.platform` as far as i know. we could use any of the 3. they are both of type `PlatformKey` so they should be the same. i'll update to use `project.platform` since that's more straightforward